tiledb/api/src/stats.rs

```diff
@@ -0,0 +1,70 @@
+use crate::error::Error;
+use crate::Result as TileDBResult;
+
+pub fn stats_enable() -> TileDBResult<()> {
+    let c_ret : i32 = unsafe {
+        ffi::tiledb_stats_enable()
+    };
+
+    if c_ret == ffi::TILEDB_OK {
+       Ok(())
+    } else {
+        Err(Error::from("TileDB stats were not successfully enabled."))
+    }
+}
+
+pub fn stats_disable() -> TileDBResult<()> {
+    let c_ret : i32 = unsafe {
+        ffi::tiledb_stats_disable()
+    };
+
+    if c_ret == ffi::TILEDB_OK {
+        Ok(())
+    } else {
+        Err(Error::from("TileDB stats were not successfully disabled."))
+    }
+}
+
+pub fn stats_reset() -> TileDBResult<()> {
+    let c_ret : i32 = unsafe {
+        ffi::tiledb_stats_reset()
+    };
+
+    if c_ret == ffi::TILEDB_OK {
+       Ok(())
+    } else {
+       Err(Error::from("TileDB stats were not successfully reset."))
+    }
+}
+
+pub fn stats_dump_str() -> TileDBResult<String> {
```

Comment:
Do you have any idea what the difference between `stats_dump_str` and `stats_dump_raw_str` is? Looking through the code paths in core I can't figure out the difference as they both appear to just end up calling the same function:

https://github.com/TileDB-Inc/TileDB/blob/dev/tiledb/sm/stats/global_stats.cc#L69-L71
https://github.com/TileDB-Inc/TileDB/blob/dev/tiledb/sm/stats/global_stats.cc#L82-L84

Unless you know of a difference I'd just wrap `ffi::tiledb_stats_dump_str` and delete the other function. Then you should rename this function to just `dump() -> TIleDBResult<String>`.